### PR TITLE
Redirection

### PR DESCRIPTION
**Pull request by Juliocho990**
### Summary
- Redirection button was added to go to the book list view.

### Screenshot
Back button

### Trello card
https://trello.com/c/59EhoFGY/33-detalle-de-libro-vi-redirecci%C3%B3n

### CODE REVIEW

File: scss/list.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
   display: flex;
   flex-grow: 1;
   justify-content: center;
-  margin-top: 70px;
+  // margin-top: 70px;
```
Comment (CrisVega08) [inline scss/list.scss]: deleted comment

File: detail.html [hunk 2]
```diff
@@ -9,6 +9,9 @@
 </head>
 <body class="body">
   <div class="container">
+    <div class="redirection">
+      <a class="redirection__button" href="./list.html">Atrás</a>
```
Comment (CrisVega08) [inline detail.html]:
```suggestion
      <a class="redirect__button" href="./list.html">Atrás</a>
```

File: detail.html [hunk 3]
```diff
@@ -9,6 +9,9 @@
 </head>
 <body class="body">
   <div class="container">
+    <div class="redirection">
```
Comment (CrisVega08) [inline detail.html]:
```suggestion
    <div class="redirect">
```

File: scss/detail.scss [hunk 4]
```diff
@@ -6,6 +6,23 @@
   justify-content: center;
 }
 
+.redirection {
```
Comment (CrisVega08) [inline scss/detail.scss]: redirect

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/list.scss (hunk 1 above)
Comment (Juliocho990) [reply scss/list.scss]: ok

File: scss/detail.scss (hunk 4 above)
Comment (Juliocho990) [reply scss/detail.scss]: ok

**Review by eledwinn — APPROVED** [review]

**Pull request merged**